x-pack/plugins/security_solution/public/resolver/store/data/reducer.test.ts

```diff
@@ -22,14 +23,46 @@ describe('Resolver Data Middleware', () => {
   });
 
   describe('when data was received and the ancestry and children edges had cursors', () => {
+    let firstChildNodeInTree: TreeNode;
+    let statsForFirstChild: Record<string, number>;
     beforeEach(() => {
       const generator = new EndpointDocGenerator('seed');
+      const baseTree = generator.generateTree({
+        ancestors: 1,
+        generations: 2,
+        children: 3,
+        percentWithRelated: 100,
+        alwaysGenMaxChildrenPerNode: true,
+      });
+      const { children } = baseTree;
+      firstChildNodeInTree = [...children.values()][0];
+      statsForFirstChild = firstChildNodeInTree.relatedEvents.reduce(
```

Comment:
I'm still reading through this file, but the `beforeEach` is long now. I'm reading it top to bottom, so I'm curious what `statsForFirstChild` is, and what it is used for.
A comment explaining the purpose of it would be nice.
Also, the `beforeEach` is long enough that you could consider moving pieces of it into helper functions. Perhaps `statsForFirstChild(treeNode: TreeNode): Record<string, number>`?

Reply:
https://github.com/elastic/kibana/pull/70097/commits/99aef33a25756499fe6d6420b545675284a5a2fa